charts/server/templates/statefulset.yaml

```diff
@@ -94,6 +94,10 @@ spec:
             - secretRef:
                 name: {{ . }}
             {{- end }}
+            {{- if .Values.woodpecker.metrics }}
```

Comment:
I think it would be better if this would be a top-level option. I see no need to nest it under `woodpecker`.